ui/components/app/flask/icons/verified-icon.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function VerifiedIcon({ className, color = '#4CD964' }) {
```

Comment:
I guess this is unrelated to the rest of the PR? We don't use this icon anywhere yet. Which is good, since we don't have any verified snaps yet.

Reply:
Yes, I saw it part of the design and decided to just make the icon now, since it technically will be a part of the settings page at some point.